templates/kubernetes/2/rancher-compose.yml

```diff
@@ -1,8 +1,7 @@
 .catalog:
   name: "Kubernetes"
-  version: "v1.2.4-rancher6"
+  version: "v1.2.4-rancher7"
   description: "Rancher Kubernetes service"
```

Comment:
we should put a minimum rancher version as it's not supposed to work with pre 1.1 (no scale policy support prior to that)